Formula/swi-prolog.rb

```diff
@@ -31,6 +31,14 @@ class SwiProlog < Formula
   depends_on "unixodbc"
 
   def install
+    # Remove shim paths from binary files `swipl-ld` and `libswipl.so.*`
+    on_linux do
+      inreplace "cmake/Params.cmake" do |s|
+        s.gsub! "${CMAKE_C_COMPILER}", "\"gcc\""
```

Comment:
Can we use `ENV.cc` instead?  It should return the correct versioned `gcc` (normally `gcc-5`, but this will change eventually).

Reply:
I wasn't sure if that would be better in this particular situation.
As far as I can tell, the values don't seem to be used for compiling/building, but are expanded inside macros to be used at runtime.

On macOS, it ends up using `"cc"` and `"c++"`
On Windows/MinGW, it ends up using `gcc.exe` and `g++.exe`

The specific inreplace is modifying the following:
```cmake
if(APPLE)
  set(C_CC "cc")
  set(C_CXX "c++")
else()
  set(C_CC     ${CMAKE_C_COMPILER})
  set(C_CXX    ${CMAKE_CXX_COMPILER})
endif()
```